src/Features/VisualBasic/Portable/Completion/CompletionProviders/AwaitCompletionProvider.vb

```diff
@@ -42,5 +44,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
         Private Protected Overrides Function GetAsyncSupportingDeclaration(token As SyntaxToken) As SyntaxNode
             Return token.GetAncestor(Function(node) node.IsAsyncSupportedFunctionSyntax())
         End Function
+
+        Private Protected Overrides Function IsDotAwaitKeywordContext(syntaxContext As SyntaxContext, cancellationToken As CancellationToken) As Boolean
+            Return False
+        End Function
+
+        Private Protected Overrides Function GetExpressionToPlaceAwaitInFrontOf(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As SyntaxNode
+            Return Nothing
+        End Function
+
+        Private Protected Overrides Function GetDotTokenLeftOfPosition(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As SyntaxToken?
+            Return Nothing
+        End Function
```

Comment:
do you intend to support these as part of the pr?  or are you just stubbing this out for someone else to add later if interested?  thanks!

Reply:
Both is possible. Let me see how well the remaining work (awaitf) goes. If I do VB, it will probably be less sofisticated then the C# version.